index-pt-br.html

```diff
@@ -76,26 +74,62 @@
 
 <h2>Versão rápida</h2>
 
-<p>Nossa conferência é dedicada a fornecer uma experiência livre de assédio para todos, independentemente do sexo, idade, orientação sexual, deficiência, aparência física, tamanho, raça, ou religião. Nós não toleramos assédio dos participantes, sob qualquer forma. Linguagem e imagens sexuais não são adequadas em nenhum local da conferência, incluindo palestras, workshops, festas, Twitter e outros meios on-line. Os participantes da conferência que violarem estas regras podem ser convidados a se retirar ou ser expulsos <em>sem restituição</em> a critério dos organizadores da conferência.</p>
+<p>
+  Nossa conferência é dedicada a fornecer uma experiência de conferência livre de assédio para todos,
+  independentemente do sexo, identidade de gênero e expressão, idade, orientação sexual, deficiência,
+  aparência física, tamanho corporal, cor de pele, etnia ou religião (ou falta dela). Nós não toleramos o assédio
+  dos participantes da conferência, sob qualquer forma. Linguagem e imagens sexuais não são apropriadas para
+  qualquer local da conferência, incluindo palestras, workshops, festas, Twitter e outras mídias on-line.
+  Os participantes da conferência que violem estas regras pode ser punidos ou expulso da conferência
```

Comment:
Typo: "... pode**m** ser punidos ou expulso**s** da conferência ...".
Maybe use "Os participantes que violem..." to avoid repeated "conferência" in the sentence.